modules/cannops/src/dvpp_call.cpp

```diff
@@ -0,0 +1,314 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#ifndef ENABLE_DVPP_INTERFACE
```

Comment:
Do we need this check in .cpp file?

If yes, then I expecting `#endif` much closer (like modules/cannops/include/opencv2/dvpp_call.hpp)

Reply:
No, it will be deleted in next commit. Thanks for the correction